index.html

```diff
@@ -0,0 +1,247 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>My Portfolio</title>
+        <link rel="stylesheet" media="screen" href="./styles.css">
+    </head>
+
+    <body>
+        <section id="portfolio" class="project-section">
+            <div>
```

Comment:
- [ ] Your header is missing maybe you deleted it by mistake, kindly fix that issue.